mappings/net/minecraft/nbt/ChunkUpdateState.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS ccl net/minecraft/nbt/ChunkUpdateState
```

Comment:
nbt?

Reply:
Should PersistentState even be in nbt?

Reply:
Agree. The whole thing should go to world package.